src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector256.cs

```diff
@@ -675,6 +675,7 @@ static Vector256<ulong> SoftwareFallback(ulong value)
         /// <param name="e29">The value that element 29 will be initialized to.</param>
         /// <param name="e30">The value that element 30 will be initialized to.</param>
         /// <param name="e31">The value that element 31 will be initialized to.</param>
+        /// <remarks>This method corresponds to __m256i _mm256_setr_epi8</remarks>
```

Comment:
Note that these method are not Intel specific. They will correspond to other instructions on other architectures.

Reply:
@jkotas is it enough if I mark that "on Intel these instructions correspond to... "?
Where I can find a list on how you transform these to assembly code for various platforms ?

Reply:
> is it enough if I mark that "on Intel these instructions correspond to... "?

Yes, something like this would work. I would say "On x86, this method correspond to ...".

> Where I can find a list on how you transform these to assembly code for various platforms ?

x86 is the only one implemented right now. ARM64 is being worked on.